vnext/Scripts/rnw-dependencies.ps1

```diff
@@ -202,9 +255,22 @@ if (!(IsElevated)) {
     return;
 }
 
-$NeedsRerun = 0
-$Installed = 0
-foreach ($req in $requirements)
+$NeedsRerun = 0;
+$Installed = 0;
+$filteredRequirements = New-Object System.Collections.Generic.List[object]
```

Comment:
```suggestion
$filteredRequirements = $()
```

Reply:
It complained about immutable collections, but I'll try again...

Reply:
Confirmed, it keeps failing with: `Exception calling "Add" with "1" argument(s): "Collection was of a fixed size."` So keeping as-is.